ibu-imager/installation_configuration_files/services/installation-configuration.service

```diff
@@ -5,7 +5,7 @@ After=nodeip-configuration.service prepare-installation-configuration.service
 [Service]
 Type=oneshot
 RemainAfterExit=yes
-ExecStart=/usr/local/bin/installation-configuration.sh
+ExecStart=/usr/local/bin/ibu-imager post-pivot --recert-image ${RECERT_IMAGE}
```

Comment:
First we can test in while running seed and i assume we will always find the issue in case we will have it as this part will be tested on each run of the flow.
I think this is worth running without image pull

Reply:
Host os == seed rchos so we should not have issues

Reply:
But it's not host vs seed, it's binary compiler OS vs host OS

Reply:
I would say it is build container os (as far as i remember) . In anycase issue will be found pretty quickly so i don't see issues with it. Wondering how many shared libraries we will ever add.

Reply:
>In anycase issue will be found pretty quickly

I don't think this is a healthy assumption, if we know of potential time bombs we should defuse them while we have a clear idea of what we're doing, not some poor person in the future that's gonna have to dig into all of it to fix weird `ldd` error

>Wondering how many shared libraries we will ever add.

It's not us, it's Go, it's RHCOS. Try compiling a go binary on your machine and run it on some old RHEL, you'll run into missing library errors. This is one of the biggest problems containers solve